src/wp-admin/css/common.css

```diff
@@ -508,14 +508,14 @@ code {
 .widefat td ol,
 .widefat td ul {
 	font-size: 13px;
-	line-height: 1.5em;
+	line-height: 1.5;
```

Comment:
Does the line-height value will same? Could you share the screenshot before after chances?